spec/beaker/dsl/helpers/web_helpers_spec.rb

```diff
@@ -36,21 +36,43 @@ def fetch_allows
       it "returns its second and third arguments concatenated." do
         concat_path = "#{destdir}/#{name}"
         create_files([concat_path])
-        expect( logger ).to receive( :notify ).with( /^Already\ fetched\ / )
```

Comment:
It doesn't seem terribly important here, but why is the `with` rspec method removed here?